app/pages/sections/order-items/catalogue-solutions/select/routes.js

```diff
@@ -226,7 +240,6 @@ export const catalogueSolutionsSelectRoutes = (authProvider, addContext, session
       sessionManager.saveToSession({
         req, key: sessionKeys.selectedRecipients, value: selectedRecipients,
       });
-
```

Comment:
I think it is better to keep the blank line here.

Reply:
done